management/tests/commands/add-identity-to-user.test.ts

```diff
@@ -0,0 +1,87 @@
+import { Server } from "http";
+import { AddressInfo } from "net";
+
+import { addIdentityToUser } from "../../src/commands";
+import { InvalidValidationCodeError } from "../../src/errors";
+import * as checkVerificationCode from "../../src/queries/check-verification-code";
+import { getIdentityType } from "../../src/utils/get-identity-type";
+import { nonPrimaryNewIdentity } from "../mocks/identities";
+import { app } from "../mocks/test-server/server";
+
+describe("Add identity to user", () => {
+  let server: Server;
+
+  beforeAll(async () => {
+    await new Promise<void>((resolve) => {
+      server = app.listen(0, () => resolve());
+    });
```

Comment:
Giving a port value of 0 here means that it will seek for a random and empty port to launch the server, avoiding us race conditions issues between all tests. Once launched, we can access the port number with : `server.address().port`.

Reply:
Sexy af.